x-pack/test/security_solution_api_integration/test_suites/investigation/timeline/trial_license_complete_tier/tests/index.ts

```diff
@@ -8,7 +8,7 @@
 import { FtrProviderContextWithSpaces } from '../../../../../ftr_provider_context_with_spaces';
 
 export default function ({ loadTestFile }: FtrProviderContextWithSpaces) {
-  describe('@ess SecuritySolution Timeline', () => {
+  describe('@ess @serverless @skipInServerless SecuritySolution Timeline', () => {
```

Comment:
Should be reviewed in https://github.com/elastic/kibana/issues/183645#issuecomment-2244774840